tests/v2/validation/steve/vai/scripts/script.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -e
+
+# URL for the SQLite binary
+SQLITE_URL="https://raw.githubusercontent.com/brudnak/rancher/brudnak-steve-29/tests/v2/validation/steve/vai/scripts/sqlite3"
```

Comment:
I would discourage adding binaries to a source repository.

How about using a small independent go program using the same access library lasso uses instead?

As a point of inspiration, this works on my machine:


```golang
package main

import (
	"database/sql"
	"fmt"
	"log"
	"os"
	"strings"

	"github.com/pkg/errors"
	_ "modernc.org/sqlite"
)

func main() {
	tableName := strings.ReplaceAll(os.Args[1], "\"", "")
	resourceName := os.Args[2]

	db, err := sql.Open("sqlite", "/var/lib/rancher/informer_object_fields.db")
	if err != nil {
		log.Fatal(err)
	}
	defer db.Close()

	query := fmt.Sprintf("SELECT \"metadata.name\" FROM \"%s\" WHERE \"metadata.name\" = ?", tableName)
	stmt, err := db.Prepare(query)
	if err != nil {
		log.Fatal(err)
	}
	defer stmt.Close()

	var result string
	err = stmt.QueryRow(resourceName).Scan(&result)
	if err != nil {
		if errors.Is(err, sql.ErrNoRows) {
			fmt.Println("Resource not found", result)
		} else {
			log.Fatal(err)
		}
	} else {
		fmt.Println("Found resource:", result)
	}
}

```

Reply:
@moio Thanks so much for the review! 🙏 

Implemented your suggestion and removed the sqlite3 binary, replacing it with a small independent go program as you suggested. Thanks again!